docs/Reference/Release-Types.md

```diff
@@ -0,0 +1,16 @@
+---
+title: QKM release types
+---
+
+# Release types
+
+Quorum Key Manager has 2 different release types - long term support (LTS) releases and interim releases.
+
+LTS releases are reliable, production-ready releases that are updated and supported throughout the lifetime
+of the release. LTS releases are supported for 12 months and are patched with every bug fix during the
+course of its support timeline. LTS releases use the format `v<year>.<month>.0`, for example v21.12.0, and are
+marked LTS in the [release notes](https://github.com/ConsenSys/quorum-key-manager/blob/main/CHANGELOG.md).
+
+Interim releases provide access to recent features that may still be experimental and are not recommended
+for production. Use interim releases for testing and development purposes. Interim releases are published
+between LTS releases and increment the patch number, for example v21.12.1.
```

Comment:
v21.12.1 corresponds to the first pathed version of the LTS v21.12.X. As I mentioned above, there is not specific format for interim or LTS versions

Reply:
Fixed